### PR TITLE
ACA-48 - Get case assignments endpoint

### PR DESCRIPTION
**Pull request by smathangi**
Still need to add few more negative test scenarios. And unit tests for Service method.

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ACA-48

### Change description ###
Get case assignments endpoint

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/managecase/service/CaseAssignmentService.java [hunk 1]
```diff
@@ -65,6 +74,45 @@ public CaseAssignmentService(PrdRepository prdRepository,
         return policyRoles;
     }
 
+    public List<CaseAssignedUsers> getCaseAssignments(List<String> caseIds) {
+
+        List<ProfessionalUser> professionalUsers = prdRepository.findUsersByOrganisation().getUsers();
```
Comment (MSancaktutar) [inline src/main/java/uk/gov/hmcts/reform/managecase/service/CaseAssignmentService.java:79]: I'm assuming this is querying only the active users in the organisation.

File: src/main/java/uk/gov/hmcts/reform/managecase/api/controller/CaseAssignmentController.java [hunk 2]
```diff
@@ -81,6 +87,62 @@ public CaseAssignmentResponse assignAccessWithinOrganisation(
...
+                                            + "        \"last_name\": \"Roberts\",\n"
+                                            + "        \"email\": \"bill.roberts@greatbrsolicitors.co.uk\",\n"
+                                            + "        \"case_roles\": [\n"
+                                            + "          \"[Collaborator]\",\n"
+                                            + "          \"[Creator]\"\n"
+                                            + "        ]\n"
+                                            + "      }\n"
+                                            + "    ]\n"
+                                            + "  }\n"
+                                            + "]",
```
Comment (danlysiak) [inline src/main/java/uk/gov/hmcts/reform/managecase/api/controller/CaseAssignmentController.java:120]: Is this example rendered as the correct expected response in Swagger? Looks like the response is an object where the array is within `case_assignments`?

Comment (MSancaktutar) [reply src/main/java/uk/gov/hmcts/reform/managecase/api/controller/CaseAssignmentController.java:120]: Well, I missed that point, thank you Dan. Perhaps generating the JSON String from a constant object with set attributes could be a better way to maintain complex example values like this.

Comment (smathangi) [reply src/main/java/uk/gov/hmcts/reform/managecase/api/controller/CaseAssignmentController.java]: Good catch Dan. Thanks! Pushed updated doc. 
Mutlu - Swagger accepts a String value only for that @Example field.

Comment (MSancaktutar) [reply src/main/java/uk/gov/hmcts/reform/managecase/api/controller/CaseAssignmentController.java]: @smathangi  - Of course, but what I mean was - we could get that String from a static method call that serialises a constant example object, couldn't we?

Comment (MSancaktutar) [reply src/main/java/uk/gov/hmcts/reform/managecase/api/controller/CaseAssignmentController.java]: Ahh, that's an annotation parameter, which would accept only a constant value. A programmatic solution is still possible, but with a little dirty work-around, and not worth discussing in this particular context. Ok, ignore my comment @smathangi .

**Pull request merged**